src/features/projectsV2/ProjectDetails/components/ImageCarousel.tsx

```diff
@@ -3,9 +3,10 @@ import Stories from 'react-insta-stories';
 import getImageUrl from '../../../../utils/getImageURL';
 import { SliderImage } from '../../../projects/components/PlantLocation/ImageSlider';
 import { SingleCarouselImage } from './microComponents/SingleCarouselImage';
+import { PlantLocationCoordinate } from '../../../common/types/plantLocation';
 
 interface Props {
-  images: SliderImage[];
+  images: SliderImage[] | PlantLocationCoordinate[] | undefined;
```

Comment:
Why was the type updated?

Reply:
PlantLocationCoordinate[] has been removed